components/CodeSharePost.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Link from 'next/link';
+
+// helpers
+import { getDate } from '@helpers/data-formatter';
+
+// Models
+import { codesharePost } from '@models/CodeShare.model';
```

Comment:
```suggestion
import { CodeSharePost } from '@models/CodeShare.model';
```

Interfaces should be TitleClase for convention.